tests/spec/fs.write.spec.js

```diff
@@ -65,4 +65,49 @@ describe('fs.write', function() {
       });
     });
   });
+
+  it('should fail to write to a file that does not exist', function(done) {
+    const fs = util.fs();
+    const buffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
+
+    fs.write('/myfile', buffer, 0, buffer.length, 0, function(error) {
+      expect(error).to.exist;
+      expect(undefined).to.equal(undefined);
```

Comment:
This assertion doesn't really add much.  What if we modify it to check the type of error we get back?  For example:

```
expect(error.code).to.equal('ENOENT');
```